src/main/java/mops/foren/domainmodel/User.java

```diff
@@ -1,10 +1,18 @@
 package mops.foren.domainmodel;
 
-import java.util.HashMap;
+import java.util.Collection;
 
 public class User {
```

Comment:
Is User supposed to be a parent class of all the different role classes ?
If so why don`t the role classes extend the user class ?